### PR TITLE
chore: ensures all fuzz tests are part of the workspace

### PR DESCRIPTION
**Pull request by miraclx**
Context: https://github.com/near/nearcore/pull/5769

### CODE REVIEW

**Review by khorolets — APPROVED** [review]

File: Cargo.lock [hunk 1]
```diff
@@ -4361,6 +4426,15 @@ dependencies = [
  "serde_json",
 ]
 
+[[package]]
+name = "rust-base58"
+version = "0.0.4"
```
Comment (matklad) [inline Cargo.lock:4431]: Where dose this come from? Seems like a weird dependency.

Comment (miraclx) [reply Cargo.lock:4431]: `near-jsonrpc-fuzz`: https://github.com/near/nearcore/blob/376791e6783aa72cfd93665770a9fa865a04df69/chain/jsonrpc/fuzz/fuzz_targets/fuzz_target_1.rs#L126

Comment (matklad) [reply Cargo.lock:4431]: We use `bs58` elsewhere, let's update the fuzz tests to use that as well?

**Comment by mina86** [comment]
By the way, we might want to have a `scripts/check_fuzz.py` or something which like `check_pytest.py` and `check_nightly.py` would check if all fuzz tests are included in nightly.

**Review by pmnoxx — APPROVED** [review]
LGTM

**Review by matklad — APPROVED** [review]

**Pull request merged**